src/cascadia/TerminalApp/AppCommandlineArgs.h

```diff
@@ -41,15 +41,32 @@ class TerminalApp::AppCommandlineArgs final
     static const std::wregex _commandDelimiterRegex;
 
     CLI::App _app{ "wt - the Windows Terminal" };
+
+    // This is a helper struct to encapsulate all the options for a subcommand
+    // that produces a NewTerminalArgs.
+    struct NewTerminalSubcommand
+    {
+        CLI::App* subcommand;

```

Comment:
I don't 100% understand how a subcommand has-a subcommand, but I am willing to accept it